### PR TITLE
Add LRS wrapper to optimizer

### LINKED ISSUE #1167: 如何设置learning rate decay？Annealing the learning rate
在[optimizers.py](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/trainer_config_helpers/optimizers.py#L360)里面，我看到了设置learning rate decay的参数， 不过有两个，learning_rate_decay_a=0.,
learning_rate_decay_b=0.,
请问这两个参数有什么区别呢、分别代表什么含义呢？我应该用哪一个呢？
似乎并没有相关的wiki、文档记录呢。

### PR DESCRIPTION
**Pull request by reyoung**
Fix #1167

### CODE REVIEW

**Comment by luotao1** [comment]
感谢您给PaddlePaddle贡献代码。由于Paddle V1/V2版本已不再维护，相关代码也已从develop分支上删除，因此关闭您的PR，欢迎您向Paddle最新版-Fluid贡献代码。
Thanks for contributing to PaddlePaddle! Since V1/V2 will not be maintained anymore, and related codes have been deleted from develop branch as well, we close this PR. Welcome to contribute to Fluid——the latest version of PaddlePaddle.
